frontend/eda/rulebook-activations/RulebookActivationForm.tsx

```diff
@@ -215,7 +233,37 @@ export function RulebookActivationInputs() {
         isRequired
         labelHelp={t('Rulebooks will be shown according to the project selected.')}
         labelHelpTitle={t('Rulebook')}
+        additionalControls={
+          <Button
+            variant="link"
+            data-cy={'manage_event_stream'}
+            isDisabled={!rulebook}
+            onClick={() =>
+              setDialog(
+                <SourceEventStreamMappingModal
+                  rulebook={rulebook}
+                  mappings={sourceMappings}
+                  setSourceMappings={setSourceMappings}
+                />
+              )
+            }
+          >{`${t('Manage event streams')}`}</Button>
```

Comment:
Small nitpick: We can remove the extra curly brackets and backticks here `{t('Manage event streams')}`